lib/manager/helmv3/update.ts

```diff
@@ -20,7 +21,7 @@ export function bumpPackageVersion(
     }
     logger.debug({ newChartVersion });
     bumpedContent = content.replace(
-      /^(?<version>version:\s*).*$/m, // TODO #12070
+      regEx(`^(?<version>version:\\s*).*$`, 'm'),
```

Comment:
```suggestion
      regEx(`^(?<version>version:\\s*).*$`, 'm'), // TODO #12071
```

I think this should be module scope